db-service/lib/cqn2sql.js

```diff
@@ -130,7 +130,7 @@ class CQN2SQLRenderer {
 
   SELECT(q) {
     let { from, expand, where, groupBy, having, orderBy, limit, one, distinct, localized } = q.SELECT
-    if (!expand) expand = q.SELECT.expand = has_expands(q) || has_arrays(q)
+    expand = q.SELECT.expand = expand ?? (has_expands(q) || has_arrays(q))
```

Comment:
Please avoid this `foo = foo || bar` pattern and prefer the `if (!foo) foo = bar` one